browser/ui/sidebar/sidebar_model.cc

```diff
@@ -91,8 +91,9 @@ void SidebarModel::AddItem(const SidebarItem& item,
   }
```

Comment:
I think notifying `OnItemAdded` before updating the active index could be potential bug. Can we notify it after updating index?

Reply:
As we have another observer interface `OnActiveIndexChanged()`, I think the order is not important.

Reply:
Ah.. I know what you mean. I think we should update active index before firing `OnItemAdded()`. and then we should fire `OnActiveIndexChanged()`. I'll handle it as a f/u.